db/version_set.cc

```diff
@@ -3573,7 +3573,7 @@ uint64_t VersionStorageInfo::EstimateLiveDataSize() const {
   // sst files with no range overlap in same or higher level. The less
   // compacted, the more optimistic (smaller) this estimate is. Also,
   // for multiple sorted runs within a level, file order will matter.
-  uint64_t size = 0;
+  uint64_t non_blob_data_size = 0;
```

Comment:
I think we could keep accumulating everything into `size`, that is there's no need for separate `non_blob` and `blob` sizes since we're going to add them up in the end anyways.